dbt/models/iasworld/schema/iasworld.addn.yml

```diff
@@ -20,20 +27,48 @@ sources:
             description: '{{ doc("column_areasum") }}'
           - name: card
             description: '{{ doc("column_card") }}'
+            tests:
+              - not_null:
+                  name: iasworld_addn_card_not_null
+              - dbt_utils.accepted_range:
+                  name: iasworld_addn_card_gte_1
+                  min_value: 1
           - name: cdu
             description: '{{ doc("column_cdu") }}'
           - name: chgrsn
             description: '{{ doc("column_chgrsn") }}'
           - name: class
             description: '{{ doc("shared_column_class") }}'
+            tests:
+              - relationships:
+                  name: iasworld_addn_class_in_ccao_class_dict
+                  to: source('ccao', 'class_dict')
+                  field: class_code
+                  config:
+                    where: |
+                      taxyr >= '2022'
+                      AND class != 'EX'
+                      AND cur = 'Y'
+                      AND deactivat IS NULL
```

Comment:
This test is on basically every `class` column in iasWorld now and is designed to catch issues like people adding a dash to classes, e.g. `2-12`.